lib/autoproj/reporter.rb

```diff
@@ -49,8 +49,25 @@ def error(error)
                 Autoproj.message line, STDERR
             end
         end
+
+        def reset_timer
+            @timer_start = Time.now
+        end
+
+        def elapsed_time
+            return nil unless @timer_start
+            secs = Time.now - @timer_start
+            [[60, :seconds], [60, :minutes], [24, :hours], [1000, :days]].map do |count, name|
```

Comment:
I think you might want to read more about [Time#strftime](https://ruby-doc.org/core-2.2.0/Time.html#method-i-strftime)

Reply:
Right ... you can't if yout want to achieve the same result (that is, not showing the time periods that are irrelevant). Sorry for the noise.

Reply:
No problem... :+1: